tpg.py

```diff
@@ -52,7 +52,17 @@
 __url__ = 'http://cdelord.fr/tpg/'
 
 import re
-import sre_parse
+import warnings
+try:
+    warnings.filterwarnings('error')
+    import sre_parse
+
+except DeprecationWarning:
+    sre_parse = re._parser
+
+finally:
+    warnings.resetwarnings()
+
```

Comment:
Wouldn't it be simpler to first check for `re._parser` and then import `sre_parse`?

```suggestion
try:
    sre_parser = re._parser
except AttributeError:
    import sre_parser
```